exercises/F-logical-operators/exercise.js

```diff
@@ -11,14 +11,32 @@ var cssLevel = 4;
 
 // Finish the statement to check whether HTML, CSS knowledge are above 5
 // (hint: use the comparison operator from before)
-var htmlLevelAbove5;
-var cssLevelAbove5;
+if(htmlLevel>5){
```

Comment:
you can try and shorten you code to look a tiny bit cleaner by doing something like this:
`let htmlLevelAbove5 = htmlLevel > 5;`
the program is going to check automatically if that statement is true or not and return the respective value.
The same goes for all the statements in this file.